.github/workflows/release.yaml

```diff
@@ -0,0 +1,61 @@
+name: Release
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
```

Comment:
@lukashass Isn't it sufficient to only do it for tags because the import steps are only executed on tags

Reply:
@mariusheine It's basically testing itself (everything that doesn't actually release it)